src/pyOpenMS/pyopenms/_parquet_query.py

```
@@ -0,0 +1,246 @@
+    def _consume_connector(self):
+        if self._condition_count == 0:
+            return ""
+        if self._pending_connector is None:
+            return "AND"
+        connector = self._pending_connector
+        self._pending_connector = None
+        return connector
+
+    def _quote_string(self, value):
+        if value is None:
+            raise ValueError("String filter value cannot be None")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)